website/gatsby-browser.js

```diff
@@ -13,11 +13,12 @@ exports.onInitialClientRender = function () {
   }
   if (window) {
     window.jQuery =
-      window.jQuery ||
-      (() => ({
-        change: () => {},
-        trigger: () => {},
-      }))
+      window.jQuery || function(nodeOrSelector) {
+        if (typeof(nodeOrSelector) == 'string') {
+            return document.querySelector(s);
+        }
+        return nodeOrSelector;
```

Comment:
It would be good to document the hack here in a comment.

Also, do we not run Prettier and ESLint on this repo? This looks formatted differently and uses old function style 🤔

Reply:
@felixfbecker it looks like we are 3.0.3 sourcegraph/prettierrc
 is this the latest? 
 should we switch over to gatsby's plugin: https://www.gatsbyjs.com/plugins/gatsby-plugin-prettier-eslint/ ?

Reply:
There are a lot of prettier updates, I'll do this in a separate PR.